nvtabular/workflow.py

```diff
@@ -664,7 +664,9 @@ def apply(
         output_path="./ds_export",
         output_format="parquet",
         out_files_per_proc=None,
-        **kwargs,
+        num_io_threads=0,
+        nsplits=None,  # Deprecated
+        num_out_files=None,  # Deprecated
```

Comment:
Is nsplits or num_out_files used anywhere? Can we just remove them rather than mark as deprecated?

Reply:
Sure - I was still allowing users to pass these args but was complaining about it in a warning.  I am happy to remove them altogether now since so much of the api is changing anyway :)